spec/models/study_topic_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+
+RSpec.describe StudyTopic, type: :model do
+  # Columns
+  it { is_expected.to have_db_column(:name).of_type(:text)
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.